docs/guides/data-management.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Data Management
+description: Complete guide on typical data management operations in ReductStore.
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+    <link rel="canonical" href="https://www.reduct.store/docs/guides/data-managment"/>
+</head>
+
+# Data Management With ReductStore
+
+ReductStore provides a set of methods to manage data in the database. This guide provides an overview of typical data management operations in ReductStore and explains how to perform them using the ReductStore SDKs or the HTTP API.
+
+## Changing Labels
```

Comment:
The other guides start with a Concepts section, which on this page can be used to more or less explain what is coming in the future, otherwise it looks like a guide to changing labels instead of data management.

Reply:
@AnthonyCvn n You're right. I've planned the data removal section as a separate task.  We use the concept section mostly to explain what is under the hood of the database. I don't see anything valuable here for this section.  Let's leave it as it is and I'll improve it when I add the removal section.